### ISSUE TITLE
Ability to have a flow without Resubmit and Resume buttons

### ISSUE THREAD

**Issue by GoutsmitSam** [invictus-integration/docs-ifa#204]
[Pim Simons]
At several of our customers I have some flows that either don't require the Resubmit and Resume buttons or the flow implementation does not support it (no Logic Apps). Archiveis still used to archive manually handled error instances.
 
I usually just explain that for these flows the Resubmit and Resume cannot be used and the users should ignore them. But I think it would be better if we could hide these buttons for these flows. For example by introducing an extra setting under the Advanced Settings of a flow like this: 

![Image](https://github.com/invictus-integration/docs-ifa/assets/127121297/6af69d1b-df83-4623-861a-dcd826e1aaa4)

**Comment by stijnmoreels** [comment]
@pim-simons , a quick question: would it be enough that the flows showing in the Dashboard UI that are not originated from LogicApps don't have those Resubmit/Resume buttons - without an additional toggle switch? That would already make more sense in any case. Hiding those buttons manually seems like - besides a bit of clanky functionality - also a root for a lot of 'where are my buttons'-support problems.

**Comment by pim-simons** [comment]
> [@pim-simons](https://github.com/pim-simons) , a quick question: would it be enough that the flows showing in the Dashboard UI that are not originated from LogicApps don't have those Resubmit/Resume buttons - without an additional toggle switch? That would already make more sense in any case. Hiding those buttons manually seems like - besides a bit of clanky functionality - also a root for a lot of 'where are my buttons'-support problems.

That sounds a lot better indeed. So if the flows are not originating from Logic App do not show the Resubmit and Resume buttons, except for if the `Custom Resubmit URL` settings is enabled 😉

**Comment by GoutsmitSam** [comment]
@pim-simons @stijnmoreels Well, the feature has already been developed in the meantime :)

Having said that, I'd like to keep it as it is though, because I have heard of scenario's where you want to hide the resubmit/resume of Logic Apps as well. 

We can look into if we want and can optimize it further, for example by auto-hiding them for Function traces

**Comment by pim-simons** [comment]
> Having said that, I'd like to keep it as it is though, because I have heard of scenario's where you want to hide the resubmit/resume of Logic Apps as well, so it's nice to have the option to do so.

That is a fair point as well 👍🏻

**Comment by stijnmoreels** [comment]
> Having said that, I'd like to keep it as it is though, because I have heard of scenario's where you want to hide the resubmit/resume of Logic Apps as well, so it's nice to have the option to do so.

Has it be considered that this will also have a support impact? As this will result in people disabling things and that they can't find it anymore.

Also: this wasn't what the original issue here was about, it was more about how flows were incorrectly showing those buttons. This is not by any means an 'optimization' but actually a bug.

**Comment by GoutsmitSam** [comment]
@stijnmoreels  I have to trust that users are intelligent enough to remember they turned it off. :)

As for the 'original issue', I'm going with the original description of how Pim put it (start of this issue), but I also seem to remember someone else mentioning such a need

[Pim Simons]
**_At several of our customers I have some flows that either don't require the Resubmit and Resume buttons or the flow implementation does not support it (no Logic Apps). Archiveis still used to archive manually handled error instances.

I usually just explain that for these flows the Resubmit and Resume cannot be used and the users should ignore them. But I think it would be better if we could hide these buttons for these flows. For example by introducing an extra setting under the Advanced Settings of a flow like this:_**

**Comment by stijnmoreels** [comment]
Yes, exactly, the original issue's description makes my point, that it's about a bug, not an improvement or feature request.

**Comment by MichaelMorozovCodit** [comment]
So what's the verdict? I don't mind discarding this and starting anew!

**Comment by pim-simons** [comment]
I think it is a bit of both honestly. 
Flows showing things they cannot support 👉🏻  bug
Being able to not show the buttons on a flow that does support them 👉🏻 feature request

When you look at my original description it has a mention of both (flows that do support it but do not require it, and flow implementation does not support it).

> At several of our customers I have some flows that either don't require the Resubmit and Resume buttons or the flow implementation does not support it (no Logic Apps).

Either way, I am fine with the current implementation as a first version 👍🏻 
We can always look at improving on this later on

**Comment by GoutsmitSam** [comment]
All right, while I do understand @stijnmoreels 's point, we will leave the implementation as is for now, and evaluate later on if we need to change or refine it.

Thanks for the feedback, all!
